frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardGraph.tsx

```diff
@@ -45,13 +42,18 @@ const MonitoringDashboardGraph: React.FC<MonitoringDashboardGraphProps> = ({
           </PrometheusGraphLink>
         </>
       ) : (
-        <Area
-          title={title}
-          humanize={humanize}
-          byteDataType={byteDataType}
-          namespace={namespace}
-          query={query}
-        />
+        <>
+          <h5>{title}</h5>
+          <PrometheusGraphLink query={query}>
+            <QueryBrowser
+              hideControls
+              defaultTimespan={defaultTimespan}
+              namespace={namespace}
+              queries={[query]}
+              isStack
+            />
+          </PrometheusGraphLink>
+        </>
```

Comment:
@vikram-raj   can we avoid repetition of `PrometheusGraphLink` and `QueryBrowser` and just change `isStack` to true or false

Reply:
Done.